recipes/zserio/all/conanfile.py

```diff
@@ -25,12 +26,24 @@ class ZserioConanFile(ConanFile):
         "fPIC": True
     }
 
+    @property
+    def _compilers_minimum_version(self):
+        # https://github.com/ndsev/zserio/tree/master/compiler/extensions/cpp#supported-compilers
+        return {
+            "apple-clang": "11",
+            "clang": "11",
+            "gcc": "7",
```

Comment:
C++11 is supported since GCC 4.7 (not all features), the GCC 7 supports C++14 as well. Can we drop the minimal compiler version required, or does it require C++14?

Reply:
Hm .. I probably forgot to allow gcc 5 after the fix with minimum C++ standard version. However our [documentation](https://github.com/ndsev/zserio/tree/v2.13.0/compiler/extensions/cpp) for first Zserio version in Conan states that it supports gcc 5.4. Thus I would like to change it at least to `"gcc" :  "5"`.

Or is it supposed that it is user's problem when older gcc version that the version supported by library is used? The other question is what should I do if Zserio 2.14.0 build fails with gcc 5 in your pipelines (note that we now support [gcc 7.4.0+](https://github.com/ndsev/zserio/tree/v2.13.0/compiler/extensions/cpp))

Reply:
changed to gcc 5 for now